.ci/runTutorial.m

```diff
@@ -0,0 +1,24 @@
+function runTutorial(tutorialName)
+
+% include the root folder and all subfolders
+addpath(genpath(pwd))
```

Comment:
Is it necessary to add the entire path? What about hidden folders? 

Unless I am mistaken, it should be enough to only add the root itself, i.e. `addpath(pwd)`. The remaining paths are added when running `initCobraToolbox`.